app/main.py

```diff
@@ -1,3 +1,12 @@
 def cache(func):
-    # Write your code here
-    pass
+    dic = {}
```

Comment:
Could you rename this variable pls? Name `dic` doesn't say anything about what is stored in it.